packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx

```diff
@@ -34,25 +39,36 @@ export function getStyles(props: ISearchBoxStyleProps): ISearchBoxStyles {
             border: '1px solid WindowText'
           },
           ':hover': {
-            borderColor: palette.neutralDark,
+            borderColor: semanticColors.inputBorderHovered,
             selectors: {
               [HighContrastSelector]: {
                 borderColor: 'Highlight'
               }
             }
           },
-          ':hover $iconContainer': {
-            color: palette.themeDark
+          ':hover .ms-SearchBox-iconContainer': {
```

Comment:
Could you also as part of this PR extract this static classNames in an object at top of the file as we do for most of our components and then use them in your selectors. Not blocking bu would be nice to bring this control to the pattern we have set up for rest of the components. Thanks!

Reply:
@Vitalius1 You bet, thanks!